build/testdata/bundles/mysql/porter.yaml

```diff
@@ -58,6 +58,7 @@ upgrade:
       name: "{{ bundle.parameters.mysql-name }}"
       namespace: "{{ bundle.parameters.namespace }}"
       chart: stable/mysql
+      version: 0.10.2
```

Comment:
I like picking a specific chart version for both install and upgrade to avoid picking up unexpected changes like this. 👍